packages/@react-aria/overlays/src/usePopover.ts

```diff
@@ -74,23 +80,25 @@ export function usePopover(props: AriaPopoverProps, state: OverlayTriggerState):
   let {
     triggerRef,
     popoverRef,
+    groupRef,
     isNonModal,
     isKeyboardDismissDisabled,
     shouldCloseOnInteractOutside,
     ...otherProps
   } = props;
 
+  let isSubmenu = otherProps['trigger'] === 'SubmenuTrigger' || otherProps['trigger'] === 'SubDialogTrigger';
```

Comment:
This is kind of a hack. Should we add a prop to override `isDismissable` even when `isNonModal` is true? Changing it to true all the time mostly works, but it breaks RSP v3 submenus which use a different DOM structure. We could refactor that to match RAC but it would be a bit more work.